Core/Core/Transports/SQLite.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Data;
```

Comment:
Unnecessary Reference using.

Reply:
Please revert SQLite to the version on the dev branch - any unnecessary change leads to unnecessary checks